File: PHP Excel Examples/CellsApiDemo/Column/groupColumns.php
<?php

use Spire\Cloud\Excel\Sdk\Api\CellsApi;
use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new CellsApi($configuration);

$name = "GroupColumns.xlsx";
$folder = "input";
$storage = null;
$sheetName = "Sheet4";
$firstIndex = 1;
$lastIndex = 4;
$hide = true;
$apiInstance->groupColumns($name, $sheetName, $firstIndex, $lastIndex, $hide, $folder, $storage);

File: Python Excel Examples/CellsApiDemo/column/groupColumns.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration
from spirecloudexcel.api.cells_api import CellsApi

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.cells_api.CellsApi(configuration)
name = "GroupColumns_1.xlsx"
sheetName = "Sheet4"
first_index = 1
last_index = 4
hide = "true"
storage = ""
folder = "/Cells/Column/"
api.group_columns(name, sheet_name=sheetName, first_index=first_index,
                       last_index=last_index, hide=hide,
                       folder=folder,
                       storage=storage)
